tests/test_events/test_events_ratios.py

```diff
@@ -47,3 +51,25 @@ def test_event_checkin_and_cancellation_ratio_per_room(self):
             event_ratio_per_room_query,
             event_ratio_per_room_response
         )
+
+    def test_event_checkin_and_cancellation_single_room(self):
```

Comment:
Nice work @jesseinit could you work in the word ratio to the name of this test, as it is conveys something different from what it's doing
`test_event_checkin_and_cancellation_ratio_single_room`

Reply:
@kwanj-k I think the `ratio` word pretty much describes what is happening under the hood. In my honest opinion, its only fair to leave it like this - going by how it has been done - as one can make sense of those portion of the code by reading the docstring above or even the name for the test case.